ui/ui.js

```diff
@@ -189,6 +189,8 @@ shaka.ui.Overlay = class {
       ],
       addSeekBar: true,
       addBigPlayButton: false,
+      fastForwardOnTaps: true,
+      rewindOnTaps: true,
```

Comment:
I think we should default this options to be false, as in some cases, having new functionalities available by default could generate conflicts on existing additional elements of the UI that are currently being created externally by applications that use the library.

This way, the feature is available to developers only if they consciously decide to add it into their projects! 

Let me know what do you think about this!

Reply:
@nbcl, I agree. I was going to change it before getting merged. There will be more changes to make so instead of constantly making it false and true while testing changes, I assigned it as true. If the changes are good I'll have to add tests and documentation, so I'll make these changes during that commit. 
Thank you.